test/debug-caller-test.js

```diff
@@ -97,8 +99,43 @@ describe("The debugCaller library", function() {
             });
 
             it("should assign random colors", function() {
-                expect(logger.log.color).toBeUndefined();
-                expect(logger.error.color).toBeUndefined();
```

Comment:
These tests fail on the current version at master. When randomColors is set to TRUE we should expect color toBeDefined.